README.md

```diff
@@ -1 +1,194 @@
-# book-an-appointment
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly add a "tests" section on your readme with specific instructions about how to correctly run your tests.